.gitignore

```
@@ -5,3 +5,4 @@
 .idea/
 /vendor
 /htdocs
+node_modules
```

Nit: Could you end this file with a newline to make future diffs nicer?
```suggestion
node_modules

```